api/cpp/CMakeLists.txt

```diff
@@ -304,3 +304,5 @@ if(BUILD_TESTING)
     slint_test(eventloop)
     target_link_libraries(test_eventloop PRIVATE Threads::Threads)
 endif()
+
+install(FILES README.md DESTINATION .)
```

Comment:
This also installs the README into the `CMAKE_INSTALL_PREFIX` - that seems very different to me how from-source built cmake packages typically look like.

I think the README should only be included in the top-level directory of the binary package. Perhaps this can be done using `CPACK_INSTALL_COMMANDS`?

```
set(CPACK_INSTALL_COMMANDS "${CMAKE_COMMAND} -E copy ${CMAKE_CURRENT_SOURCE_DIR}/README.md $ENV{CMAKE_INSTALL_PREFIX}")
```

(untested!)

Reply:
I will drop this part for now! Adding extra files into CPack is harder than I expected.

The recommended way is to install the file in question in a separate component and then make cpack include that component in its install. Unfortunately I always end up with the install target _also_ installing the extra component:-(